mypy/checker.py

```diff
@@ -2890,6 +2890,8 @@ def check_protocol_variance(self, defn: ClassDef) -> None:
             if not isinstance(tvar, TypeVarType):
                 # Variance of TypeVarTuple and ParamSpec is underspecified by PEPs.
                 continue
+            if tvar.values:
```

Comment:
so the `TypeVarTypes.values` are the constraints or something?

Reply:
yeah, apparently its legacy stuff

```py
T = TypeVar("T", values=(int, str))
```

Reply:
the name `constraints` makes no sense whatsoever, so I think I'd actually prefer `values` 🤷🏻

Reply:
why didn't they just name it `choices` or something?